corn-maze.lic

```diff
@@ -0,0 +1,632 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#corn-maze
+  This script runs non-combat tasks for sleeping dragon maze. 
+=end
+
+custom_require.call(%w(common drinfomon textsubs common-travel events common-items))
```

Comment:
is textsubs needed here?

Reply:
why the hell?

Uh, I'm not sure.  All I can say is that is was in task-loot originally.  Why though?

Reply:
I can't think of a single reason that should be there.  I'll unload textsubs and run it again tomorrow though, before I say definitively.

Reply:
yeah there's no reason, it's safe to remove. just using this excuse to actually look over the script now that you're submitting it and noticing anything stupid that was already there :D